lib/matplotlib/axes/_base.py

```diff
@@ -4195,9 +4226,9 @@ def twiny(self):
         return ax2
 
     def get_shared_x_axes(self):
-        """Return a copy of the shared axes Grouper object for x axes."""
+        'Return a reference to the shared axes Grouper object for x axes'
```

Comment:
This is actually less compliant with most of the tools I know, including the lack of period. Is this an intentional change?

Reply:
The change from "copy of" to "reference to" is a correction I made.  I  have reverted the other formatting changes, although my preference would actually be to use single double-quotes for these.  I don't know why I used single-quotes.

Reply:
For the record, I like the single double- (or even single- !) quote too. I just know I've had to push away from those to make some checkers happy. (They're good to catch other problems.)